README.md

```diff
@@ -10,6 +10,12 @@ You can install yuyo from PyPI using the following command.
 python -m pip install -U hikari-yuyo
 ```
 
+The following feature flags ensure feature-specific optional dependencies are also installed:
+
+* `hikari-yuyo[asgi]` ensures the dependencies required to run the Asgi RESTBot adapter.
+* `hikari-yuyo[sake]` can be used to ensure the installed Sake version is compatible Yuyo's
+  functionality which can use Sake.
```

Comment:
Add a note that it's recommended you still have your own pinned Sake (and maybe Hikari) versions as this doesn't pin